README.md

```diff
@@ -4,336 +4,231 @@
 
 ### Nightfall_Code_Scanner - a code review tool that protects you from committing sensitive information
 
-nightfall_code_scanner scans your code for secrets or sensitive information. It’s intended to be used as a part of your CI to simplify the development process, improve your
+The `nightfall_code_scanner` is a code review tool that protects you from committing sensitive information to your
+version control. It is intended to be used as a part of your CI to simplify the development process, improve your
 security, and ensure you never accidentally leak secrets or other sensitive information via an accidental commit.
 
 ## Supported Services
+* [GitHub Action](https://github.com/nightfallai/nightfall_dlp_action)
+* [CircleCI Orb](https://github.com/nightfallai/nightfall_circle_orb)
 
-### GithubActions
+## NightfallDLP Config File
```

Comment:
should we add the default here at the top?

Reply:
done